tests/suites/test_suite_ecp.function

```diff
@@ -16,6 +16,44 @@
     mbedtls_ecp_point_free( x );    \
     mbedtls_ecp_point_init( x );
 
+/* Auxiliary function to compare two mbedtls_ecp_group objects. */
+inline static int mbedtls_ecp_group_cmp( mbedtls_ecp_group *P,
```

Comment:
Very minor: P and Q feel weird to me as names for groups / curves. May I suggest grp1 and grp2?

Reply:
Fixed as suggested.